added comments to logistic regression example in the documentation

andreh7 opened the pull request:
I'm proposing a few additional comments in the logistic regression example (affecting theano/tutorial/examples.html ). No code changes.

nouiz commented on doc/tutorial/examples.txt:
```diff
@@ -489,16 +489,27 @@ It will be used repeatedly.
+    # between training iterations (updates)
     w = theano.shared(rng.randn(feats), name="w")
+
+    # initialize the bias (constant) term
```
Why did you put "(constant)" ? Otherwise, this PR is good to merge. thanks.

andreh7 commented:
updated the branch, removed "(constant)" in commit 6253c08 . 

Originally I had put 'constant' since the result of the bias term is not proportional to any input, similar to the constant term in a polynomial. But I agree, the way it was written was a bit misleading, one could think that the bias term is fixed and not allowed to vary during training.

nouiz commented:
thanks
